src/devtools/store.js

```diff
@@ -711,6 +729,26 @@ export default class Store extends EventEmitter {
           // The profiler UI uses them lazily in order to generate the tree.
           i = i + 3;
           break;
+        case TREE_OPERATION_UPDATE_SUSPENSE_STATE:
```

Comment:
What is the "debugger tab?

Reply:
I'm a little bummed to see another fiber-type-specific tree operation being added here. I haven't had time to review this yet or think through it fully, so maybe it's appropriate, but I hope we don't need to do much more of this.

Reply:
>What is the "debugger tab"?

The Suspense Debugger tab. The thing I'm adding in this PR. :-)

>I'm a little bummed to see another fiber-type-specific tree operation being added here.

I'm not sure how else I can read the current Suspense state from inside the tab. The information that's in the `store` right now is insufficient because it only tells me about children (and no info about individual child). The only way to get the current state right now is with `useInspectedElement` but that only works for one. I want to get the Suspense state of all Suspense nodes in the tree.

Reply:
I thought about it similarly to Profiler operations. We definitely have Profiler-specific events. This seems similar to me. I don't think about these events as being "specific to a type of fiber", I think about them as "things that happen in Suspense world", analogous to "things that happen in Profiler world".

I don't really care though. Just trying to get it working.

Reply:
> The Suspense Debugger tab. The thing I'm adding in this PR. :-)

Oh 😆 right.

> We definitely have Profiler-specific events (well, one).

True. Arguably that shouldn't be sent with the tree operations either. It's just convenient to batch it with the other work.

I'm wondering if we should add a new message for things like this, so the tree operations are purely focused on the tree. I think this might help simplify the fact that multiple things need to care about tree operations (`Store` and `CommitTreeBuilder`) but I think maybe nothing outside of the Suspense debugger tab cares about this new operation?

I'll give it some more thought when I sit down to do a real review. I was just kind of thinking out loud with my earlier comment I guess. 😄

Reply:
> maybe nothing outside of the Suspense debugger tab cares about this new operation?

That should be correct.